### PR TITLE
Make sure `HostCollectiveId` is set for Organizations

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/6616

[add-funds.webm](https://user-images.githubusercontent.com/12435965/228704769-84fe0584-5658-41f7-aaec-23bad7ce0bf6.webm)

### CODE REVIEW

File: server/graphql/v2/object/AccountPermissions.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const AccountPermissions = new GraphQLObjectType({
     },
     addFunds: {
       type: new GraphQLNonNull(Permission),
-      description: 'Whether the current user can mark this order as expired',
+      description: 'Whether the current has teh permission to add funds to the account',
```
Comment (znarf) [inline server/graphql/v2/object/AccountPermissions.ts:19]:
```suggestion
      description: 'Whether the current has permission to add funds to the account',
```

File: migrations/20230329091450-set-hostCollectiveId-in-collectives.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+      UPDATE "Collectives"
+      SET
+        "HostCollectiveId" = id
+      WHERE "isHostAccount" = true
+      AND "type" = 'ORGANIZATION'
+      AND "HostCollectiveId" IS NULL
```
Comment (znarf) [inline migrations/20230329091450-set-hostCollectiveId-in-collectives.js:12]:
```suggestion
      AND "isActive" IS TRUE
      AND "approvedAt" IS NOT NULL
```

**Review by znarf — CHANGES_REQUESTED** [review]
Assumptions needs to be rechecked.

**Comment by SudharakaP** [comment]
> Assumptions needs to be rechecked.

Yes, my mistake. I was under the impression that after making a host one should be able to add funds. But it seems that only after activating the budget the host is able to add funds and receive financial contributions etc.

**Pull request closed without merging**